src/main/java/org/opendatakit/suitcase/model/SyncClientError.java

```diff
@@ -0,0 +1,28 @@
+package org.opendatakit.suitcase.model;
+
+public class SyncClientError {
+    private static final int UNKNOWN_ERROR_STATUS_CODE = 500;
+    private final int statusCode;
+    private final String message;
+
+    public SyncClientError(Exception e) {
+        String errorMsg = e.getCause().getMessage();
+        int startIndex = errorMsg.indexOf("Status");
+        if (startIndex != -1) {
+            statusCode = Integer.parseInt(errorMsg.substring(startIndex + 7, startIndex + 10));
```

Comment:
You can add comments about the use of startIndex + 7, startIndex + 13, or one sample error message. So that it will be easier in the future to understand code.